modules/persistent-storage-local-create-machine-config.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-local.adoc
+
+[id="local-create-machine-config_{context}"]
+= Configure an additional storage device to use local volumes
+
+You can configure additional storage devices for local volumes by defining new machineconfig resources.
+
+.Prerequisites
+
+* The Local Storage Operator is installed.
+* Local disks are attached to the {product-title} nodes.
+
+.Procedure
+
+. Create an `ifcfg` file that defines the `HWADDR` corresponding to the MAC address of the storage device to be configured.
```

Comment:
I did not include the file example because it was for a network adapter. But here it is:
```
HWADDR=52:54:00:73:4f:32
TYPE=Ethernet
BOOTPROTO=none
IPADDR=192.168.100.102
PREFIX=24
DNS1=192.168.100.254
ONBOOT=yes
GATEWAY=192.168.100.254
```
Can these key-value pairs be modified for a storage device?